flow/project.py

```diff
@@ -1748,6 +1818,7 @@ def _get_operations_status(self, job, cached_status):
         for key in sorted(status_dict):
             yield key, status_dict[key]
 
+    @deprecated(deprecated_in="0.11", removed_in="0.13", current_version=__version__)
```

Comment:
What should a user use in future versions? Or are we removing this functionality from the public API as well?

Reply:
@mikemhenry I don't have any problem in keeping this function but if we keep this, we should warn a user that it's not implemented for an aggregate operation (`len(jobs) > 1`).
This is because we're now storing information per aggregate and not a single job in #336

Reply:
I think we should potentially make `_get_group_status` public and encourage users to switch to that.

Reply:
@b-butler We are deprecating most of the public methods, I don't find a strong reason introducing this method in the API. Will we be later deprecating `get_group_status`?

Reply:
One more reason I found to not to make it public is that it will never return the "complete" status of the jobs as labels are also included in a status for a job. This returns only the status of a job per operation hence I think that we'd then have to make both `_get_job_labels` and `_get_group_status` public.

Reply:
We could keep this public now with static aggregation, but we would need to accept aggregates as well.